src/PromoSection/index.js

```diff
@@ -10,7 +11,7 @@ function goToDocGetStarted() {
 
 function goToFeatures() {
   logEvent('promo', 'features')
-  window.location = '/features'
+  Router.push('/features')
```

Comment:
minor: put an empty line like in get-started .. or remove it everywhere :)

Reply:
Done

Reply:
👍